tests/monster_test_ps.fbs

```diff
@@ -0,0 +1,10 @@
+// similar schema, except to be parsed with --prefix-size
+
+include "monster_test.fbs";
```

Comment:
I'd prefer it if we keep the test simple without needing yet another schema. You could simply generate a size-prefixed buffer in code, and then see if you can access it correctly.

Reply:
I understand that, I have thought about it similarly. However, I'm not quite sure how else I may check the interoperability between C++ and Java (in this case), if not for a generated file. One could add another dedicated test that checks for interoperability in all supported languages? This test would first generate the regular and the size-prefixed version, and then invoke a small test for each language. This way, the new schema would not need to be versioned in git.

Reply:
I'm not sure that an intra-language test is that important, since all we'd be checking is if a language uses the right size (32 bits)..